ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -765,6 +1049,26 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
         break;
       }
 
+      // Check if op is supported by backend
+      bool is_supported = false;
+
+      // Create nGraph backend
+      string ng_backend_type;
+      BackendManager::GetCurrentlySetBackendName(&ng_backend_type);
+      // Create backend to query
+      Status status = BackendManager::CreateBackend(ng_backend_type);
+      ng::runtime::Backend* op_backend =
+          BackendManager::GetBackend(ng_backend_type);
+      TF_RETURN_IF_ERROR(IsSupportedByBackend(node, op_backend, TFtoNgraphOpMap,
+                                              is_supported));
+      BackendManager::ReleaseBackend(ng_backend_type);
```

Comment:
Maybe create backend once and check for all nodes and then release, instead of creating-releasing for each node